src/ophyd_async/planstubs/fly.py

```diff
@@ -0,0 +1,113 @@
+from typing import List
+
+import bluesky.plan_stubs as bps
+
+from ophyd_async.core.detector import DetectorTrigger, StandardDetector, TriggerInfo
+from ophyd_async.core.flyer import HardwareTriggeredFlyable
+from ophyd_async.core.utils import in_micros
+from ophyd_async.panda._table import SeqTable, SeqTableRow, seq_table_from_rows
+from ophyd_async.panda._trigger import SeqTableInfo
+
+
+def fly_and_collect(
```

Comment:
This should probably have a more specific name.

Something like `time_resolved_trigger_detectors`?

Reply:
I agree, I just wanted to call it something and figured i'd ask for opinions before merging.